pom.xml

```diff
@@ -78,7 +78,7 @@
         <aether.version>0.9.0.M2</aether.version>
         <apache.curator.version>4.1.0</apache.curator.version>
         <apache.curator.test.version>2.12.0</apache.curator.test.version>
-        <apache.kafka.version>2.1.1</apache.kafka.version>
+        <apache.kafka.version>2.2.2</apache.kafka.version>
```

Comment:
Can you also update extensions-core/kafka-indexing-service to use this version as well please

I had a PR to consolidate the kafka version, but never got around to fully testing it - https://github.com/apache/druid/pull/9117/files

Reply:
Ah, missed that it still had it's own version defined, removed.